### PR TITLE
[Trivial] Add pricegraph README

### PR DESCRIPTION
**Pull request by nlordell**
Added a README with some instructions on building and testing the Wasm bindings for the price graph.

### Test Plan

Try out the commands in the "Building" section in the README.

### CODE REVIEW

**Review by e00E — APPROVED** [review]
Thank you Nick. Works on my machine.

File: pricegraph/wasm/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+can be used from JS environments.
+
+## Usage
+
+```js
+const estimator = new PriceEstimator(encodedOrders);
+const [WETH, DAI] = [1, 7];
+console.log(estimator.estimatePrice(WETH, DAI, 100 * 10e18));
+esimator.free();
+```
```
Comment (bh2smith) [inline pricegraph/wasm/README.md:13]: So is this from a node env? Do I have to import `PriceEstimator` or something?

**Review by bh2smith — APPROVED** [review]
Very nice, concise and straightforward readme. I successfully ran the integration test and generate pkg examples here, but am confused about the first.

File: pricegraph/wasm/README.md (hunk 1 above)
Comment (nlordell) [reply pricegraph/wasm/README.md]: Added a bit more context, let me know if this is better.

File: pricegraph/wasm/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+This crate provides WebAssembly bindings to the `pricegraph` crate so that it
+can be used from JS environments.
+
+## Usage
+
+The Wasm bindings can be used from JavaScript or TypeScript by importing the
+generated NodeJS module:
+
+```js
+import { PriceEstimator } from "@gnosis.pm/dex-pricegraph";
```
Comment (bh2smith) [inline pricegraph/wasm/README.md:12]: Nice! I really like this!

File: pricegraph/wasm/README.md (hunk 1 above)
Comment (bh2smith) [reply pricegraph/wasm/README.md]: Nice thanks!

**Pull request merged**